ButteryToast/Toast.swift

```diff
@@ -91,20 +92,39 @@ open class Toast: Operation {
     topVC.view.addSubview(self.view)
     
     let yOffset: CGFloat
-    if #available(iOS 11.0, *) {
-      yOffset = topVC.view.safeAreaInsets.top
-    } else if topVC.navigationController != nil {
-      yOffset = topVC.topLayoutGuide.length
-    } else {
-      yOffset = UIApplication.shared.statusBarFrame.size.height
+    
+    var orientation: Orientation = .top
```

Comment:
better to guarantee immutability (if you can) - so I would declare with `let` then set to .top in an else case with the assertion failure we had before.